repo2docker/buildpacks/r.py

```diff
@@ -243,6 +248,7 @@ def get_build_scripts(self):
                     r"""
                     apt-get update && \
                     apt-get install --yes r-base={} && \
+                    apt-get install --yes r-base-dev={} && \
```

Comment:
We also need `libclang-dev` to add it I'd replace these lines with:
```
                    apt-get install --yes r-base={R_version} \
                         r-base-dev={R_version} \
                         libclang-dev && \
```
and then edit the
```
                    """.format(
                        self.r_version
                    ),
```
a few lines later to read:
```
                    """.format(
                        R_version=self.r_version
                    ),
```

Reply:
thank you!